tests/test_cluster_read.py

```diff
@@ -29,6 +29,18 @@ def test_date(self):
         self.assertGreaterEqual(int(v), before)
         self.assertGreaterEqual(after, int(v))
 
+    def test_count(self):
+        string = """
+            - name: print.output
+              log_source_command: echo -e "error line_1log_line4"
+              log_regexp_error: '"level":"error"'
+              log_regexp_warning: '"level":"warning"'
+        """
+        ri = opl.cluster_read.RequestedInfo(string)
+        k, v = next(ri)
+        self.assertEqual(k, "print.output")
+        self.assertEqual(v, {"all": 1, "error": 1})
```

Comment:
```suggestion
        self.assertEqual(v, {"all": 4, "error": 2, "warning": 1})
```

Reply:
If you commit mine suggestions to the test (and if I have not made any error in it), test will guide you to the right solution.